test/fragment-cases/trailing-linebreaks.html

```diff
@@ -1 +1 @@
-trailing linebreaks are allowed
+trailing linebreaks are allowed
```

Comment:
Will need to edit the .editorconfig to remove the add-newline-at-end-of-file rule for this file